pkg/version/version.go

```diff
@@ -45,3 +47,17 @@ func Get() Info {
 func (info Info) String() string {
 	return info.GitVersion
 }
+
+func init() {
+	buildInfo := prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "kubernetes_build_info",
+			Help: "A metric with a constant '1' value labeled by major, minor, git version, git commit and git tree state from which Kubernetes was built.",
+		},
+		[]string{"major", "minor", "gitVersion", "gitCommit", "gitTreeState"},
+	)
+	info := Get()
+	buildInfo.WithLabelValues(info.Major, info.Minor, info.GitVersion, info.GitCommit, info.GitTreeState).Set(1)
+
+	prometheus.MustRegister(buildInfo)
```

Comment:
One small nit -- shouldn't you register the metric first before setting a value?